test/webrtc_test.rb

```diff
@@ -113,15 +123,26 @@ def configure_and_start()
         deploy("task_a", "task_b", Time.at(5), "127.0.0.1:3012")
         configure_and_start
 
-        data_in = Types.iodrivers_base.RawPacket.new
-        data_in.time = Time.now
-        data_in.data = [1, 0, 1, 0, 1, 1, 1, 0]
+        output = expect_execution do
+            task1.stop!
+            task2.stop!
+        end.to do
+            emit task1.stop_event
+            emit task2.stop_event
+            [have_one_new_sample(task1.status_port),
+             have_one_new_sample(task2.status_port)]
+        end
 
-        data = expect_execution do
-            syskit_write task1.data_in_port, data_in
-        end.to { have_one_new_sample task2.data_out_port }
+        assert_equal :NoDataChannel, output[0].data_channel
+        assert_equal :NoDataChannel, output[1].data_channel
```

Comment:
You are already testing that stops closes the data channel in another test ... no need to repeat it here.

These are *unit tests*. Keep things clear by testing what you mean to test (which here is it "successfully reestablishes connection after a stop/start cycle") and not more.